src/components/cylc/Task.vue

```diff
@@ -180,6 +196,12 @@
                 @include outline();
             }
 
+            &.ready, &.queued {
```

Comment:
```suggestion
            &.submitting {
```
Is there some reason not to do this now?  (I thought the CSS classes could reflect the icon name - "submitting" - and the server-side task states (ready and queued, but may change) could be mapped separately to the icons?)

Reply:
Not sure how we want to go about this at the Cylc Flow end, might take a while before we see the "submitting" status coming through over the API.

Reply:
We don't need to do that.  The server still has to distinguish the two finer-grained states (currently "ready" and "queued") that get combined at the UI into "submitting". So I think the server should just put its true states in the workflow summary data, and the UI can do what it wants with those (which we've decided is: map both server states into "submitting").

Reply:
Simple solution!

Might lead to some documentation issues e.g. when the suite log says that a task is `waiting => queued`, or `cylc dump` output, DB, etc.

Reply:
This conversation is now somewhat out-of-date due to the different task statuses that we finally settled on.